src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -48,13 +48,18 @@
 
 using CalamaresUtils::yamlToVariant;
 
+
+const char* NetInstallPage::HEADER_TEXT = "<h1>Package Selection</h1>";
+
+
 NetInstallPage::NetInstallPage( QWidget* parent )
     : QWidget( parent )
     , ui( new Ui::Page_NetInst )
     , m_networkManager( this )
     , m_groups( nullptr )
 {
     ui->setupUi( this );
+    ui->header->setText( tr( HEADER_TEXT ) );
```

Comment:
Why not change that in the UI file already?

Reply:
this is also used as prettyName() in the view